python/feathub/processors/flink/table_builder/sliding_window_utils.py

```diff
@@ -202,6 +212,15 @@ def evaluate_sliding_window_transform(
             ],
         )
 
+    if empty_window_output_enabled:
```

Comment:
In case `empty_window_output_enabled == false`, we still want to allow users to skip emitting the same window values, right?

Could you provide the pointer to the code that enables this choice?

Reply:
I think we prevent setting `enable_static_window_output` to `False`, which skip emitting the same window values when `enable_empty_window_output` is `False`. Because with this configuration, it is impossible to determine if a value is expired or not. It is checked when FeathubClient gets the sdk_config in config.py:55.